server-plugins/notification-resources/src/index.ts

```diff
@@ -98,6 +98,43 @@ export async function OnBacklinkCreate (tx: Tx, control: TriggerControl): Promis
   return res
 }
 
+/**
+ * @public
+ */
+export async function OnDmCreate (tx: Tx, control: TriggerControl): Promise<Tx[]> {
+  const hierarchy = control.hierarchy
+  const ptx = tx as TxCreateDoc<DirectMessage>
+  const res: Tx[] = []
+
+  if (tx.createdBy == null) return []
+  if (!isDmCreationTx(ptx, hierarchy)) return []
```

Comment:
You can check it as trigger model params

Reply:
Fixed, thanks